rpc/src/rpc_types.rs

```diff
@@ -283,19 +283,42 @@ pub struct TransactionRecipient {
     pub amount: u64,
 }
 
-#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+/// The crawled known network and measurements.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct NetworkGraph {
-    pub vertices: HashSet<Vertice>,
-    pub edges: HashSet<Edge>,
```

Comment:
This change won't impact the uniqueness of the collections' values, since they are constructed in `NetworkMetrics` with `HashSet`s. Using `Vec` here instead, removes the requirement for the `Vertices` and `Edges` to implement `Hash` and `Eq`.